app/domain/logs.rb

```diff
@@ -203,11 +203,6 @@ module AuthnK8s
         code: "CONJ00037D"
       )
 
-      ValidatingHostId = ::Util::TrackableLogMessageClass.new(
```

Comment:
same

Reply:
It is not used.
I didn't find any reference to this that needs update.